secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsTool.java

```diff
@@ -0,0 +1,33 @@
+package de.adorsys.android.securemobilepush.sms;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.RequiresApi;
+import android.support.v4.content.ContextCompat;
+
+public class SmsTool {
```

Comment:
make class final and convert to util class with private constructor and 

`public static void requestSMSPermission(@Nonnull Activity activity)`